newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -11,6 +11,8 @@ import styles from './summary.module.scss';
 export default function SummaryPage() {
   const [submitting, setSubmitting] = useState(false);
   const [error, setError] = useState('');
+  const [mailSending, setMailSending] = useState(false);
+  const [mailSent, setMailSent] = useState(false);
```

Comment:
Actually, now that I see, we could probably merge these two states into one: `mailState = null | sending | sent`

Reply:
I don't use the submitting state now that I went for useBackend :)